docs/core/event_handler/bedrock_agents.md

```diff
@@ -0,0 +1,330 @@
+|-----------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------|:-------------:|:-------------:|
+| [Lambda Function](#your-first-agent)                                                                                  | Defines your business logic for the action group                               |       ✅       |       ✅       |
+| [OpenAPI Schema](#generating-openapi-schemas)                                                                         | API description, structure, and action group parameters                        |       ❌       |       ✅       |
+| Bedrock [Service Role](https://docs.aws.amazon.com/bedrock/latest/userguide/agents-permissions.html){target="_blank"} | Allows Amazon Bedrock to invoke foundation models                              |       ✅       |       ✅       |
+| Agent for Bedrock                                                                                                     | The service that will combine all the above to create the conversational agent |       ❌       |       ✅       |
+
+=== "Using AWS Serverless Application Model (SAM)"
+	Using [AWS SAM](https://aws.amazon.com/serverless/sam/){target="_blank"} you can create your Lambda function and the necessary permissions. However, you still have to create your Agent for Amazon Bedrock [using the AWS console](https://docs.aws.amazon.com/bedrock/latest/userguide/agents-create.html){target="_blank"}.
+
+	```yaml hl_lines="18 26 34 61"
+	--8<-- "examples/event_handler_bedrock_agents/sam/template.yaml"
+	```
+
+	1. Amazon Bedrock needs permissions to invoke this Lambda function
+	2. Check the [supported foundational models](https://docs.aws.amazon.com/bedrock/latest/userguide/agents-supported.html){target="_blank"}
+	3. You need the role ARN when creating the Agent for Amazon Bedrock
+
+=== "Using AWS Cloud Developer Kit (CDK)"
+	This example uses the [Generative AI CDK constructs](https://awslabs.github.io/generative-ai-cdk-constructs/src/cdk-lib/bedrock/#agents){target="_blank"} to create your Agent with [AWS CDK](https://aws.amazon.com/cdk/){target="_blank"}.
+	These constructs abstract the underlying permission setup and code bundling of your Lambda function.
+
+	```python
+    --8<-- "examples/event_handler_bedrock_agents/cdk/bedrock_agent_stack.py"
+	```
+
+	1. The path to your Lambda function handler
+	2. The path to the OpenAPI schema describing your API
+
+### Your first Agent
+
+To create an Agent for Amazon Bedrock, use the `BedrockAgentResolver` to annotate your actions.
+This is similar to the way [all the other Event Handler](api_gateway.md) resolvers work.
+
+It's required to include a `description` for each API endpoint and input parameter. This will improve the understanding Amazon Bedrock has of your actions.
```

Comment:
The tone changes mid-way through the sentence "it" & "you"

> You are required to add a `description` parameter in each endpoint, doing so will improve the Bedrock's understanding of your actions.

Also, it's not clear what an input parameter is in this context as the example doesn't show one beyond the URL

Reply:
Thanks! Changed the sentence. The example already highlights what an input parameter is.

<img width="1119" alt="image" src="https://github.com/aws-powertools/powertools-lambda-python/assets/10713/487df872-ea61-4d9f-8c2d-ef6ad7fb8dba">